File: Flick+CoreDataProperties.m
//
//  Flick+CoreDataProperties.m
//  VirtualTourist-ObC
//
//  Created by Online Training on 8/17/17.
//  Copyright © 2017 Mitch Salcido. All rights reserved.
//

#import "Flick+CoreDataProperties.h"

@implementation Flick (CoreDataProperties)

+ (NSFetchRequest<Flick *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Flick"];
}

@dynamic imageData;
@dynamic urlString;
@dynamic title;
@dynamic pin;

@end

File: Flick+CoreDataProperties.swift
//
//  Flick+CoreDataProperties.swift
//  VirtualTourist-ObC
//
//  Created by Online Training on 8/13/17.
//  Copyright © 2017 Mitch Salcido. All rights reserved.
//

import Foundation
import CoreData


extension Flick {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Flick> {
        return NSFetchRequest<Flick>(entityName: "Flick")
    }

    @NSManaged public var urlString: String?
    @NSManaged public var imageData: NSData?
    @NSManaged public var pin: Pin?

}
